docs/packages/connect/methods/getDeviceState.md

```diff
@@ -0,0 +1,44 @@
+# Get Device State
+
+Return information about a specific device
+
+```javascript
+const result = await TrezorConnect.getDeviceState(params);
+```
+
+### Params
+
+[\***\*Optional common params\*\***](commonParams.md)
+
+## Example
+
+Get the state of a device
+
+```javascript
+const capturedDevice = await TrezorConnect.getDeviceState({
+    device: {
+        instance: 0, // identificator for wallet A
+        state: undefined, // by using undefined you are making sure that passphrase will be requested
+    },
+});
+
+// Get the public key without prompting for the passphrase again
+await TrezorConnect.getPublicKey({
+    device: {
+        instance: 0,
+        state: capturedDevice.payload.state, // by using a previously capture state identifier, passphrase will not be re-requested
+    },
```

Comment:
```suggestion
    device: {
        instance: 0,
        state: capturedDevice.payload.state, // by using a previously capture state identifier, passphrase will not be re-requested
    },
    path: "m/44'/0'/0'",
    coin: "btc",
```

lets provide all required parametrs of getPublicKey method, example will not work without them :)